test/SlowTests/Server/Replication/ReplicationAutomaticConflictResolution.cs

```diff
@@ -474,6 +474,12 @@ public async Task ResolveConflictToTombstone(Options options)
 
                 Assert.True(WaitForDocument(third, "marker2$users/1"));
 
+                using (var session = first.OpenSession())
```

Comment:
you are checking that on first, but not waiting anywhere for that to be applied by first?

Reply:
Exactly! I am guessing that caused the original failure, so I want to see if next time we fail there :-)